cleanlab_studio/studio/studio.py

```diff
@@ -436,6 +436,29 @@ def TLM(
             self._api_key, quality_preset, options=options, timeout=timeout, verbose=verbose
         )
 
+    def TLMHybrid(
+        self,
+        response_model: str,
```

Comment:
```suggestion
        response_model: str = "gpt-4o",
```

Reply:
Let's have a default, so this whole thing works with: 
`tlm = studio.TLMHybrid()`

Our tutorial should still explicitly demonstrate setting specific response/scoring model